src/goto-diff/goto_diff_parse_options.h

```diff
@@ -23,7 +23,8 @@ class optionst;
   "(json-ui)" \
   "(show-goto-functions)" \
   "(verbosity):(version)" \
-  "u(unified)(change-impact)"
+  "u(unified)(change-impact)(forward-impact)(backward-impact)" \
+  "(demo-output)"
```

Comment:
I'm not sure "demo-output" is a good choice: command-line options are for eternity, make sure they are almost self-explanatory.